### PR TITLE
Support read_attribute method

### PR DESCRIPTION
**Pull request by ppworks**
Some gems([like a enumerize](https://github.com/brainspec/enumerize/blob/master/lib/enumerize/attribute.rb#L87-L89)) expects #read_attribute method.

### CODE REVIEW

**Comment by kbrock** [comment]
@zilkey would it make sense to just rename `_read_attribute` to `read_attribute`?

**Comment by kbrock** [comment]
It looks like defining the method should pick that up?

Won't [enumerize](https://github.com/brainspec/enumerize/blob/master/lib/enumerize/attribute.rb#L85-L86) pick up a method that has that defined?

**Comment by kbrock** [comment]
Also, would it make sense to use `alias_method` instead?

Think that behaves better when there are methods defined in subclasses. And this stuff is meant to be subclassed.

File: lib/active_hash/base.rb [hunk 1]
```diff
@@ -422,6 +422,7 @@ def [](key)
     def _read_attribute(key)
       attributes[key]
     end
+    alias_method :read_attribute :_read_attribute
```
Comment (ppworks) [inline lib/active_hash/base.rb]: I used `alias_method` .

**Comment by zilkey** [comment]
Looks good to me!

**Pull request merged**